wallet-server-ui/src/app/component/about/about.component.html

```diff
@@ -36,6 +36,81 @@ <h2 translate>about.heading</h2>
       </ul>
     </div>
 
+    <div class="vspacer-2"></div>
+
+    <div class="faq">
+      <label>FAQ</label>
+      <p class="question">What is going on here?</p>
+      <p>
+        This is a bitcoin wallet that supports DLC Contracts.
+      </p>
+      <p class="question">What is a DLC Contract?</p>
+      <p>
+        A DLC Contract is an agreement between counterparties on what will happen with pooled funds based on the outcome of an event, as attested to by an oracle.
+        DLC Contracts come in two types, enumerated (enum) and numeric.
+      </p>
+      <p class="question">What is an oracle?</p>
+      <p>
+        An oracle is a trusted third party that will announce and attest to the outcome of events.
+        At the maturation time of an event, the oracle is expected to publish the outcome to the event so DLCs based on the event can resolve.
+        An example of an oracle is a cryptocurrency exchange that publishes announcements and attestations to the end-of-day price on bitcoin.
+      </p>
+      <p class="question">What is an oracle announcement?</p>
+      <p>
+        An announcement is a formal declaration of an event that will take place, to be attested to by a given oracle.
+        An announcement has an eventId to specify what event it is attesting to, a maturation time, and validation information for the oracle so the outcome can only be confirmed by that oracle.
+        An announcement includes the potential outcomes or range of potential outcomes depending on whether it is for an enum or numeric contract.
+      </p>
+      <p class="question">What is a contract info?</p>
+      <p>
+        A contract info is an Announcement with additional metadata including collateral and payout amounts.
+        A contract info is essentially a prototype Offer that you can review and change to make your own Offers.
```

Comment:
I don't know that a noob would be able to parse this. Maybe something more like a contract info contains all of the information needed to construct a contract between two parties that can be turned into a DLC